### PR TITLE
Add dev containers

### PR DESCRIPTION
**Pull request by lorlen**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,40 +1,44 @@
-version: "3.9"
+version: "3.7"
```
Comment (noFrostoo) [inline docker-compose.yml:1]: why downgrade ?

**Review by noFrostoo — COMMENTED** [review]
Why donwgrade version in docker-compose.yml?

File: docker-compose.yml (hunk 1 above)
Comment (lorlen) [reply docker-compose.yml:1]: To use the oldest possible version. The `init` argument in backend needs version 3.7, so that's what I've gone for.

Comment (noFrostoo) [reply docker-compose.yml:1]: Do you know why they removed it?

**Review by noFrostoo — APPROVED** [review]

**Pull request merged**